app/config/AppConfig.scala

```diff
@@ -53,6 +53,7 @@ class AppConfig @Inject()(
   lazy val relevantLicensesUrl = loadConfig("urls.relevantLicenses")
   lazy val serviceAvailabilityUrl = loadConfig("urls.serviceAvailability")
   lazy val customsMovementsFrontendUrl = loadConfig("urls.customsMovementsFrontend")
+  lazy val feedbackFrontendUrl = loadConfig("urls.feedbackFrontendUrl")
```

Comment:
Using the name 'feedback' here for the exist survey I think is a bit confusing as it clashes with the 'giveFeedbackLink' below (line 66, used in the phaseBanner component). Could we call this new one 'exitSurvey' to more clearly distinguish it from the header feedback link?

Reply:
@glidester bike-shedding big time, but he is actually right! It is just a shame that he did not give the same suggestion on my SFUS work :stuck_out_tongue: